### PR TITLE
Chore: Auto adds templating label in PRs

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/pr-commands.json [hunk 1]
```diff
@@ -165,5 +165,11 @@
     "matches": [ "public/app/plugins/datasource/zipkin/**/*"],
     "action": "updateLabel",
     "addLabel": "datasource/Zipkin"
+  },
+  {
+    "type": "changedfiles",
+    "matches": ["public/app/features/variables/**/*", "public/app/features/templating/**/*"],
+    "action": "updateLabel",
+    "addLabel": "area/dashboard/templating"
   }
```
Comment (hugohaggmark) [inline .github/pr-commands.json:174]: Will this replace the `area/frontend` label or add `area/dashboard/templating` label @marefr?

Comment (torkelo) [reply .github/pr-commands.json:174]: good question, can multiple rules match or just one?

Comment (hugohaggmark) [reply .github/pr-commands.json:174]: if I read this test, correctly it will just add the label https://github.com/grafana/github-actions-testrepo/pull/18

Comment (marefr) [reply .github/pr-commands.json:174]: It will add area/dashboard/templating

**Review by marefr — APPROVED** [review]
LGTM

File: .github/pr-commands.json (hunk 1 above)
Comment (marefr) [reply .github/pr-commands.json:174]: And area/frontend for example

**Pull request merged**